### PR TITLE
Rename authorization to authentication and configure general styling for inputs

### LINKED ISSUE #28: Fix authentication forms to hide password field

### PR DESCRIPTION
**Pull request by warrenshen**
Fix #28

### CODE REVIEW

File: app/assets/javascripts/components/login_form.es6.jsx [hunk 1]
```diff
@@ -20,12 +21,14 @@ class LoginForm extends React.Component {
...
         <input
+          autoFocus={true}
           placeholder={'email'}
           style={this.styles.input}>
         </input>
         <input
           placeholder={'password'}
-          style={this.styles.input}>
+          style={this.styles.input}
+          type={'password'}>
```
Comment (maxwolffe) [inline app/assets/javascripts/components/login_form.es6.jsx]: Is wrapping this in {'s just a convention? Just 'type="password"' would work as well right?

**Comment by maxwolffe** [comment]
Why are Authentication and head preferable to Athorization and header? 

Otherwise LGTM.

File: app/assets/javascripts/components/login_form.es6.jsx (hunk 1 above)
Comment (warrenshen) [inline app/assets/javascripts/components/login_form.es6.jsx]: For javascript stuff you actually do need the curly braces. But for `html` string tags you don't - that's just convention!

**Comment by warrenshen** [comment]
@maxwolffe I just think that "authentication" is a more universally used word for 'login' and 'signup' (preference decision).

**Pull request merged**